src/planwise/component/datasets.clj

```diff
@@ -18,13 +19,15 @@
   [record]
   (some-> record
           (update :import-result edn/read-string)
+          (update :import-job edn/read-string)
           (update :mappings edn/read-string)))
 
 (defn dataset->db
   [dataset]
   (some-> dataset
-          (update :import-result pr-str)
-          (update :mappings pr-str)))
+          (update-if-contains :import-result #(some-> % pr-str))
+          (update-if-contains :import-job #(some-> % pr-str))
+          (update-if-contains :mappings #(some-> % pr-str))))
```

Comment:
How about changing it to a `(update-if :import-result pr-str)`, and `update-if` runs the update only if not-nil, instead of contains?